assets/launcher/lang/sklmessages_es_MX.properties

```diff
@@ -112,25 +112,25 @@ manager.profile.menu.duplicate=Duplicar
 manager.profile.menu.delete=Eliminar
 
 manager.editor.header.title=Editar Instancia
-manager.editor.footer.button.save=Guardar
 manager.editor.option.name.name=Nombre de la Instancia
 manager.editor.option.name.placeholder=<Instancia sin nombre>
 manager.editor.option.version.name=Version
 manager.editor.option.directory.name=Carpeta del juego
 manager.editor.option.directory.placeholder=<Carpeta por defecto>
 manager.editor.option.directory.info.label=Algunos mods (no siempre) necesitan ser instalados en la carpeta .minecraft. Para usar la carpeta .minecraft
 manager.editor.option.directory.info.link=Haz click aqui
+manager.editor.footer.button.save=Guardar
 
 manager.editor.button.more=Mas opciones
 manager.editor.option.memory.name=Memoria maxima (RAM)
 manager.editor.option.compatibility.name=Modo de compatibilidad
 manager.editor.option.compatibility.enabled=Activado
 manager.editor.option.visibility.name=Visibilidad del launcher
-manager.editor.option.visibility.hide=Esconder y reabrir al cerrar el juego
-manager.editor.option.visibility.close=Cerrar cuando inicie el juego
-manager.editor.option.visibility.keep=Mantener abierto y muestra la consola del juego
-manager.editor.option.visibility.keep.info=Esta opcion no es la adecuada para PC's de bajos recursos
-manager.editor.option.jvmpath.name=Archivo JAVA
+manager.editor.option.visibility.hide=Esconder (hasta cerrar MC)
+manager.editor.option.visibility.close=Cerrar completamente
+manager.editor.option.visibility.keep=Mostrar launcher y la consola
+manager.editor.option.visibility.keep.info=NOTA: No apta para computadoras de bajos recursos
```

Comment:
Original string does not start with any `NOTE:`

Reply:
changed